ntc_templates/templates/cisco_ios_show_ipv6_access-lists.textfsm

```diff
@@ -0,0 +1,23 @@
+# Value Required,Filldown ACL_TYPE (Standard|Extended)
```

Comment:
Never used (and it is a nice information).
There is also no ACL number?

Reply:
I don't think there is an ACL type and number in Ipv6